README.md

```diff
@@ -71,6 +73,10 @@ recipes:
     - '  x'
 ```
 
+There is one option you should mention in `/plugins/LiteXpansion/config.yml` file:
+
+- `nerf-other-addons` - Enable this option will nerf Energized Solar Generator and panels in InfinityExpansion, otherwise all machines in LiteXpansion will consume double energy. This option is enabled by default.
+
```

Comment:
add the auto updater to it aswell and I will merge it

Reply:
ok, but I am wondering if you like (default: `true`) or current one.

Reply:
Default true would be nice